openmc/mgxs/groups.py

```diff
@@ -54,7 +54,7 @@ def __deepcopy__(self, memo):
     def __eq__(self, other):
         if not isinstance(other, EnergyGroups):
             return False
-        elif self.group_edges != other.group_edges:
+        elif (self.group_edges != other.group_edges).all():
```

Comment:
I remember why now - it is necessary.  The array comparison (self.group_edges != other.group_edges:) returns an array of truthiness.  We want to know if all of them are true.  So, .all() is needed.  I have to put that back in now that I remember.